internal/commands/database/show.go

```diff
@@ -128,6 +128,18 @@ func (s *showCommand) Execute(exec commands.Executor, uuid string) (output.Outpu
 	}, nil
 }
 
+func getVersion(db *upcloud.ManagedDatabase) string {
+	switch db.Type {
```

Comment:
Might be good to check if `db.Metadata` is `nil` ?

Reply:
Added nil check and test for this helper